From: =?utf-8?q?Fabian_Gr=C3=BCnbichler?= <git@fabian.gruenbichler.email>
Date: Wed, 3 Sep 2025 14:44:51 +0200
Subject: bootstrap: ignore x.py shell completion diff
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

this test is intended for development, we can ignore it for package builds.


Signed-off-by: Fabian Grünbichler <git@fabian.gruenbichler.email>
---
 src/bootstrap/src/core/build_steps/test.rs | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/bootstrap/src/core/build_steps/test.rs b/src/bootstrap/src/core/build_steps/test.rs
index 9f0d1cc..b664ca2 100644
--- a/src/bootstrap/src/core/build_steps/test.rs
+++ b/src/bootstrap/src/core/build_steps/test.rs
@@ -1197,7 +1197,8 @@ HELP: to skip test's attempt to check tidiness, pass `--skip src/tools/tidy` to
             eprintln!(
                 "x.py completions were changed; run `x.py run generate-completions` to update them"
             );
-            crate::exit!(1);
+            // Debian: we don't care about this
+            //crate::exit!(1);
         }
     }
 
